<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ptSpotListWidget" name="SpotList" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="SpotDetailsGroup" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ptCheck" name="HasMaxRadius">
        <property name="text">
         <string>Use maximum radius</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ptInput" name="MaxRadius"/>
      </item>
      <item>
       <widget class="ptInput" name="ChromaWeight"/>
      </item>
      <item>
       <widget class="ptInput" name="Threshold"/>
      </item>
      <item>
       <widget class="ptCurveWindow" name="CurveWin" native="true"/>
      </item>
      <item>
       <widget class="ptInput" name="Saturation"/>
      </item>
      <item>
       <widget class="ptCheck" name="IsAdaptiveSat">
        <property name="text">
         <string>Adaptive saturation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ptInput" name="ColorShift"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptCheck</class>
   <extends>QCheckBox</extends>
   <header>ptCheck.h</header>
  </customwidget>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
  <customwidget>
   <class>ptCurveWindow</class>
   <extends>QWidget</extends>
   <header>ptCurveWindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ptSpotListWidget</class>
   <extends>QWidget</extends>
   <header>filters/imagespot/ptSpotListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
